tests/acl/test_acl.py

```diff
@@ -148,6 +148,11 @@ def setup(duthosts, rand_selected_dut, rand_unselected_dut, tbinfo, ptfadapter):
             upstream_port_ids.append(port_id)
             upstream_port_id_to_router_mac_map[port_id] = rand_selected_dut.facts["router_mac"]
 
+    # stop garp service for single tor
+    if 'dualtor' not in tbinfo['topo']['name'] and rand_unselected_dut is None:
```

Comment:
```suggestion
    if 'dualtor' not in tbinfo['topo']['name']:
```
is the check ```rand_unselected_dut is None``` necessary?